### PR TITLE
Provide extra helpful messaging for curated packages

### PR DESCRIPTION
**Pull request by a-cool-train**
*Description of changes:*

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: pkg/curatedpackages/packagecontrollerclient.go [hunk 1]
```diff
@@ -127,9 +127,6 @@ func (pc *PackageControllerClient) CreateCredentials(ctx context.Context) error
 		return errors.New("environment variables EKSA_AWS_SECRET_ACCESS_KEY and EKSA_AWS_ACCESS_KEY_ID not provided")
 	}
```
Comment (TerryHowe) [inline pkg/curatedpackages/packagecontrollerclient.go:129]: Why is this being deleted?

**Review by TerryHowe — CHANGES_REQUESTED** [review]

File: pkg/curatedpackages/packagecontrollerclient.go (hunk 1 above)
Comment (a-cool-train) [reply pkg/curatedpackages/packagecontrollerclient.go:129]: As per my discussion with @junshun , this is not necessary anymore.

**Review by TerryHowe — APPROVED** [review]
/lgtm
/approve

**Pull request merged**